src/app/components/drawer/SavedSearches/SavedSearchActions.js

```diff
@@ -193,14 +207,14 @@ const ProjectActions = ({
       <ConfirmProceedDialog
         body={
           <TextField
-            className="project-actions__edit-title"
-            defaultValue={object.title}
-            id="project-actions__edit-title-input"
+            className="saved-search-actions__edit-title"
+            defaultValue={savedSearch.title}
+            id="saved-search-actions__edit-title-input"
             label={
               <FormattedMessage
                 defaultMessage="Title"
                 description="Label for the title input when renaming a list"
-                id="projectsComponent.title"
+                id="savedSearchActions.title"
```

Comment:
Please let's keep existring string ids as they were to avoid unnecessary re-translation work and costs.

Reply:
Done